text/0265-event-vision.md

```diff
@@ -0,0 +1,74 @@
+# Event basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to write down and agree on a few basics.
+
+### What are events?
+
+Events are a type of log which has a required event name and a specific structure which is implied by that event name.
+
+The event name and its implied structure make events much more suitable for observability compared to generic logs.
+
+Because of this, the OpenTelemetry project wants to encourage the transition from generic logs to events.
+
+### OTLP
+
+Since events are a type of log, they share the same OTLP data structure and OTLP pipeline.
+
+### API
+
+OpenTelemetry should have an Event API. This will help to promote the distinction between events and generic logs,
```

Comment:
Without trying to be pedantic, we do have an event api https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md. I guess here you want to express that we need an event api that produces events according to the semconv? https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/events.md

Reply:
@joaopgrassi i'm not sure what the distinction is that you're making. the current event API is used to create semconv-compliant events.

Reply:
it's just because the text says: "OpenTelemetry should have an Event API" so "should" there is confusing

Reply:
> we do have an event api https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md

yeah, but there isn't consensus (yet) that we should keep it

Reply:
There is already https://github.com/open-telemetry/oteps/blob/main/text/0202-events-and-logs-api.md

IMO the onus is on the nays to submit an OTEP to remove the event API rather than on the yeas to affirm it. That said, I think it's fine to keep this line. I'd just change it to "should keep its" instead of "should have an"